include/picongpu/fields/incidentField/Solver.kernel

```diff
@@ -159,8 +208,12 @@ namespace picongpu
                      */
                     float3_X coeff1, coeff2;
 
-                    //! Indices of the incidentField components for the two terms
-                    uint32_t incidentComponent1, incidentComponent2;
+                    /** Indices of the incidentField components for the two terms
+                     *
+                     * Note the intentional cross combination, this is to match the general vector update stated above.
+                     */
+                    static constexpr uint32_t incidentComponent1 = (T_axis + 2) % 3;

```

Comment:
Could it be that this is only a redefinition of `dir2`?

Reply:
Well technically yes. But I think it is more clear to have both variables, as depending on the situation one name makes more sense than the other.

Reply:
Upon further consideration, I decided to keep the `dir1, dir2` variables separate for the reasons stated above. However, I now calculate them based on `incidentComponent1, incidentComponent2` to avoid differences in how the calculation is done.